AIMSICD/build.gradle

```diff
@@ -83,23 +83,36 @@ android {
 
 dependencies {
     // DO NOT REMOVE BELOW COMMENTED-OUT CODE BEFORE ASKING!
-    //compile 'com.github.amlcurran.showcaseview:library:5.0.0'
+    // Please group dependencies for better review like below.
+    
+    // Android Support Libraries
     compile 'com.android.support:appcompat-v7:23.1.1'
+    
+    // OpenCSV (comma-separated values) parser library
+    compile 'au.com.bytecode:opencsv:2.4'
+    
+    // Easy to use Root Shell for Android Applications
+    compile 'com.github.Stericson:RootShell:d708ae1f0a'
+    
+    // SLF4J binding for the Android logger
+    // https://github.com/lp0/slf4j-android
+    compile 'eu.lp0.slf4j:slf4j-android:1.7.12-0'
+    compile 'org.slf4j:slf4j-api:1.7.12'
+    
+    // Dependency Injection for Android
     compile 'io.freefair.android-injection:injection:1.0.0-rc2'
     compile 'io.freefair.android-injection:logging:1.0.0-rc2'
     compile 'io.freefair.android-injection:okhttp:1.0.0-rc2'
-    //https://github.com/lp0/slf4j-android
-    compile 'com.github.Stericson:RootShell:d708ae1f0a'
-    compile 'org.slf4j:slf4j-api:1.7.12'
-    compile 'eu.lp0.slf4j:slf4j-android:1.7.12-0'
-    compile 'au.com.bytecode:opencsv:2.4'
+    compile 'io.freefair.android-util:logging:1.1.0'
 
-    //This git hash resolves to version 5.5
-    compile 'com.github.MKergall.osmbonuspack:OSMBonusPack:2e8bca20f7'
+    // OSMBonusPack library to interact with OpenStreetMap data
+    compile 'com.github.MKergall.osmbonuspack:OSMBonusPack:v5.6'
+    
+    // Transition library for animated transition effects
     compile 'com.github.kaichunlin.transition:core:0.9.2'
-    compile 'io.freefair.android-util:logging:1.1.0'
-    //debugCompile 'com.squareup.leakcanary:leakcanary-android:1.3.1'
```

Comment:
I'm not not sure if we might need it, but let's keep it away for now.
Im not a fan of commented-out code.
If we want to use it in future, we can re-add it in future.